packages/grafana-data/src/dataframe/StreamingDataFrame.ts

```diff
@@ -43,6 +44,18 @@ enum PushMode {
   // long
 }
 
+export type SerializedStreamingDataFrame = DataFrameDTO & {
```

Comment:
Is DataFrameDTO necessary here?  I think it is assing:
```
name?: string; 
fields: Array<FieldDTO | Field>;
```

seems better to have that explicit (I assume `fields: FieldDTO[]`)

Reply:
👍 made explicit